config-windows.md

```diff
@@ -0,0 +1,3 @@
+# Windows Application Container Configuration
+
+Windows application containers can be configured using the following properties.
```

Comment:
@wking To your question of ["Where's the beef"](https://en.wikipedia.org/wiki/Where%27s_the_beef%3F), there's not much here yet.  I would expect that #504 and similar would be added over time.  My desire with this PR it clear that it is the intent of OCI to support Linux, Solaris, and Windows.